### PR TITLE
style: Remove unused styles/classes from global.scss

### LINKED ISSUE #9012: check unused styles on global.scss
Related issue: https://github.com/kodadot/nft-gallery/issues/8071

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Closes #9012
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: assets/styles/global.scss [hunk 1]
```diff
@@ -215,23 +210,6 @@ hr {
...
-  @apply z-[999] #{!important};
-}
-
-.transak_modal {
-  @apply z-[1000] max-h-[600px] #{!important};
-}
-
-#transakOnOffRampWidget {
-  height: 100% !important;
-}
```
Comment (roiLeo) [inline assets/styles/global.scss:229]: I think we used this class

Capture d’écran 2024-02-07 à 2 01 26 PM

**Review by roiLeo — COMMENTED** [review]
double check

File: assets/styles/global.scss (hunk 1 above)
Comment (pbkompasz) [reply assets/styles/global.scss]: The 'Add Funds' modal doesn't open in production and in development this error is thrown: 
Screenshot from 2024-02-08 13-29-31

**Review by roiLeo — COMMENTED** [review]
please check conflicts

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 7.11 USD/DOT ~ 2.813 $DOT
🧗 [16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt](https://polkadot.subscan.io/account/16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt)
🔗 [0x6e1a4eaff736b85934a354fec4df8098e7ffec63de0e053987806071774454bf](https://polkadot.subscan.io/extrinsic/0x6e1a4eaff736b85934a354fec4df8098e7ffec63de0e053987806071774454bf)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**